API/SSW.Rewards.Domain/Entities/Reward.cs

```diff
@@ -9,6 +7,8 @@ public class Reward : Entity
         public string Code { get; set; }
         public string Name { get; set; }
         public int Cost { get; set; }
+        public string ImageUri { get; set; }
```

Comment:
Should this be Uri type?

Reply:
As per our conversation, this is working better as a string